### PR TITLE
82 get commentscomment idlikes

### PR DESCRIPTION
**Pull request by DLiSWE**
Get comment likes

### CODE REVIEW

**Review by raylu511 — DISMISSED** [review]

**Review by raylu511 — APPROVED** [review]

**Review by ZohaibManzoor00 — APPROVED** [review]
looks good

**Review by Matt-Tan15 — DISMISSED** [review]

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

**Review by Matt-Tan15 — APPROVED** [review]
looks good!

File: src/controllers/comment.controller.ts [hunk 1]
```diff
@@ -59,6 +59,16 @@ class CommentController {
       next(error);
     }
   };
+
+  public getLikesFromComment = async (req: Request, res: Response, next: NextFunction) => {
```
Comment (ZohaibManzoor00) [inline src/controllers/comment.controller.ts:63]: nit: if not using a parameter, add underscore before it => _req: Request

**Review by ZohaibManzoor00 — APPROVED** [review]
Looks good

**Pull request merged**